ansible/roles/etcd/defaults/main.yaml

```diff
@@ -4,6 +4,10 @@ etcd_source_type: package
 
 etcd_client_port: 2379
 etcd_peer_port: 2380
+# Remove etcd_client_legacy_port & etcd_peer_legacy_port
+# when https://github.com/contiv/netplugin/issues/323 is fixed
+etcd_client_legacy_port: 4001
```

Comment:
Take a look at line 8. I know how you feel about this legacy stuff. When https://github.com/contiv/netplugin/pull/325 gets merged and a release cut, I will go back and remove the etcd_legacy. We're in a bit of a pinch to get this in, otherwise I'd hold off for 325 to get merged.